controllers/dspipeline_params.go

```diff
@@ -646,6 +651,19 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if cfg := p.APIServer.CustomKfpLauncherConfig; cfg != "" {
+			if cm, err := util.GetConfigMap(ctx, cfg, p.Namespace, client); err != nil {
+				// If the custom kfp-launcher configmap is not available, that is OK
+				if !apierrs.IsNotFound(err) {
+					log.Error(err, fmt.Sprintf("Encountered error when attempting to fetch ConfigMap: [%s], Error: %v", cfg, err))
+					return err
```

Comment:
if it's ok, I think we try to proceed with the defaults as a fallback, just like if the user didn't specify any ConfigMap file at all.

So, don't return an error here -- log it and then keep going. If you return, all the rest of the param setting below doesn't happen.

I'd log something like "User specified a CustomKfpLauncherConfig, but it's not found. Falling back to using defaults."

Reply:
Yeah my suggestion was to throw the error, but I'm fine with the fallback approach as long as we are providing sufficient logs for users to be able to deduce why their configmap was not picked up.

Reply:
gotcha. Yeah I like falling back here :) Thanks!